# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/moz.build
# Commit: 8e609f97c413
# Full Hash: 8e609f97c413a12f7042dfe80647bdd34f0400d8
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2019-11-01 16:13:11
# Regressor Bug: 1587096
# File Overlap Count: 1
# Description:
#   Bug 1587096 - Part 1: Add JS FinalizationGroupObject and related data structures r=sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49943
# ==============================================================================

diff -r f273ec2ec0ae -r 8e609f97c413 js/src/moz.build
--- a/js/src/moz.build	Fri Nov 01 10:37:09 2019 +0000
+++ b/js/src/moz.build	Fri Nov 01 10:37:25 2019 +0000
@@ -214,6 +214,7 @@
     'builtin/Boolean.cpp',
     'builtin/DataViewObject.cpp',
     'builtin/Eval.cpp',
+    'builtin/FinalizationGroupObject.cpp',
     'builtin/JSON.cpp',
     'builtin/MapObject.cpp',
     'builtin/ModuleObject.cpp',